.github/workflows/build.yml

```diff
@@ -583,7 +580,7 @@ jobs:
           else
             EVENT_CMAKE_OPTIONS=""
           fi
-          EVENT_CMAKE_OPTIONS="$EVENT_CMAKE_OPTIONS -DMBEDTLS_ROOT_DIR=/usr/local/opt/mbedtls@2"
+          EVENT_CMAKE_OPTIONS="$EVENT_CMAKE_OPTIONS -DMBEDTLS_ROOT_DIR=`brew --prefix mbedtls@2`"
```

Comment:
BTW what is `@2` can we adopt the detection that we have in cmake as well? (can be done separately)

Reply:
The `@2` forces the version 2 instead of version 3. It was added by #1204.
If we remove it, then we would be using mbedtls 3 instead of mbedtls 2. Yes, can be updated in a separate PR.